src/cross_lib/sound/cc65/vic20/vic20_sounds.c

```diff
@@ -51,7 +50,7 @@
 	{ 
 		VIC.voice1 = freq; 
 		VIC.volume_color |= 0x08; 
-		_pause();
+		_pause(); _pause();
 		VIC.voice1 = 0x00; 
 		VIC.volume_color &= 0x00; 

```

Comment:
You want to set the volume to zero.  But, do you want to set the auxiliary color, also, to zero?
If you do -- or, if you aren't using multicolor graphics, then you should use the more efficient expression:
```c
VIC.volume_color = 0x00;
```
Otherwise, you should use:
```c
VIC.volume_color &= 0xF0;
```

Reply:
That's nice to know.  It means that the optimizer knows that it should replace `&= 0x00` with the smaller `= 0x00`.